src/examples/flows/rabbitmq/jms_send.yaml

```
@@ -0,0 +1,30 @@
+id: jms_send_rabbitmq
```

+ rename the file too to match the task name

```suggestion
id: jms_produce_rabbitmq
```